parameter/ConfigurableDomains.h

```diff
@@ -31,6 +31,8 @@
 
 #include "BinarySerializableElement.h"
 #include "Results.h"
+#include <criterion/Criteria.h>
```

Comment:
As criterion is know a separate library I though it can be good to use <> form. Is it for system headers only?